### ISSUE TITLE
If you find pyvips useful, please vote at vinta/awesome-python

### ISSUE THREAD

**Issue by jcupitt** [jcupitt/pyvips#61]
Link: 

https://github.com/vinta/awesome-python/pull/1101

It needs 20 +1s to get on the list.

**Comment by techdragon** [comment]
Since the PR got auto closed, https://github.com/vinta/awesome-python/pull/1101#event-2778907573 you may want to open a new one referencing the old one, because you did get the requisite +1s to be added, however the PR went "stale" and got auto-closed.

**Comment by jcupitt** [comment]
Good idea, new PR here: https://github.com/vinta/awesome-python/pull/1579
